recipes/doe_significant_utilization_changes/build_url.py

```diff
@@ -0,0 +1,32 @@
+import pandas as pd
+import numpy as np
+import sys
+import os
+
+do_s3_endpoint=os.environ.get('DO_S3_ENDPOINT')
```

Comment:
this is a new environmental variable that hasn't appeared before. we don't need to hide this url, it's fine to give the actual endpoint

Reply:
Also this python script is almost necessary, because we can also just load this table into sql and do the sql join, there's a pgdump version of this file available too